BackendTests/UnitTests/PopulationInNorwayTypeTests.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Linq;
+using Backend.API.Services;
+using Backend.Models.SSBPopulationStatistics.GraphQLTypes;
+using GraphQL;
+using Shouldly;
+using Xunit;
+
+namespace BackendTests.UnitTests
+{
+    public sealed class PopulationInNorwayTypeTests
+    {
+        private readonly PopulationInNorwayType objectGraph = new(new MetadataServiceMocked());
```

Comment:
Let's get an underscore on this one (love the instantiation though `new(new ...)`).